### DISCUSSION TITLE
[Request] 模型适配 One-API 支持

### DISCUSSION THREAD

**Discussion by JohnLanPeluchi** [lobehub/lobehub#284] [Ideas]
### 🥰 需求描述 | Feature Description

关于模型适配One-API已经做得很好了没必要再花功夫去适配，只需要兼容One-api的各种模型就行了。
只要把插件生态弄起来 比优化lobe-chat本身要好得多 比如有人提出文字转语音的功能其实完全可以用插件解决不用在源码上动刀子

### 🧐 解决方案 | Proposed Solution

完善插件开发 文档 市场 提供更多官方插件 以及 第三方开发插件

**Comment by arvinxx** [comment]
有些基础升级是需要的，国庆这段期间正好也适合做。接下来重点会放在 agent 上，包含市场、插件、应用这块。

**Comment by canisminor1990** [comment]
https://github.com/lobehub/lobe-chat/issues/263

**Comment by arvinxx** [comment]
关于 one-api 的兼容， #305 已支持
